### ISSUE TITLE
ERROR：the input data is insufficient for an assembly

### ISSUE THREAD

**Issue by Argonum-Clever2** [Nextomics/NextDenovo#195]
**Error message**
*Suggested seed_cutoff (genome size: 680.00Mb, expected seed depth: 45, real seed depth: 9.80): 50001 bp
[113064 ERROR] 2023-11-21 10:04:54 the input data is insufficient for an assembly.

**Config file**
[General]
job_type = local # local, slurm, sge, pbs, lsf
job_prefix = nextDenovo
task = all # all, correct, assemble
rewrite = yes # yes/no
deltmp = yes 
parallel_jobs = 20 # number of tasks used to run in parallel
input_type = raw # raw, corrected
read_type = ont # clr, ont, hifi
input_fofn = ej.fofn
workdir = /data0/stu_wangfang/software/NextDenovo

[correct_option]
genome_size = 680m # estimated genome size
sort_options = -m 20g -t 15
minimap2_options_raw = -t 8
pa_correction = 3 # number of corrected tasks used to run in parallel, each correct
correction_options = -p 15

[assemble_option]
minimap2_options_cns = -t 8 
nextgraph_options = -a 1

**Operating system**
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	CentOS
Description:	CentOS Linux release 7.9.2009 (Core)
Release:	7.9.2009
Codename:	Core

**GCC**
gcc version 4.8.5 20150623 (Red Hat 4.8.5-44) (GCC)

**Python**
Python 3.9.12

**Comment by moold** [comment]
As the log says, the input data is insufficient for an  assembly, so please input  more data.

**Comment by Argonum-Clever2** [comment]
NextDenovo开发团队：
你好！
&nbsp; &nbsp; &nbsp; 我输入了在你们公司测序得到的ONT数据，但是为什么他会显示数据不可用呢？您的意思是需要输入很多的数据吗？






Argonum
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Nextomics/NextDenovo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月22日(星期三) 下午5:10
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Nextomics/NextDenovo] ERROR：the input data is insufficient for an assembly (Issue #195)





 
As the log says, the input data is insufficient for an  assembly, so please input  more data.

**Comment by moold** [comment]
Yes, but if you want to get a more accurate answer, you have to  follow [here](https://github.com/Nextomics/NextDenovo/issues/new?assignees=&labels=&projects=&template=bug-or-error-report.md&title=) to provide more details.
